source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ReleaseAtNTP.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Crown copyright
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.v2.characteristics.adms;
+
+/**
+ * Emission parameters given at actual temperature and pressure or normal temperature and pressure (1atm, 273.15K): 0=actual,1=NTP
+ */
+public enum ReleaseAtNTP {
```

Comment:
Seems a bit weird name for this enum, since one of the possible values is NTP.
What does NTP stand for? `Normal Temperature and Pressure?

Reply:
The name was taken from the `.apl` file configuration, but agreed it's not very clear. Will change it to: `ReleaseTemperatureAndPressure`.